pcsx2-qt/MainWindow.cpp

```diff
@@ -2867,11 +2867,16 @@ void MainWindow::doDiscChange(CDVD_SourceType source, const QString& path)
 	bool reset_system = false;
 	if (!m_was_disc_change_request)
 	{
-		const int choice = QMessageBox::question(this, tr("Confirm Disc Change"),
-			tr("Do you want to swap discs or boot the new image (via system reset)?"), tr("Swap Disc"), tr("Reset"), tr("Cancel"), 0, 2);
-		if (choice == 2)
-			return;
-		reset_system = (choice != 0);
+		QMessageBox messageBox(QMessageBox::Question, tr("Confirm Disc Change"),
```

Comment:
Nitpick: follow the same naming convention as the rest of the function (`snake_case`), using `camelCase` here is just inconsistent.

Reply:
Yeah, I was looking at some Qt documentation as I was writing this, and automatically used the same naming format (and one of the names) as the examples. I can fix that. The irritating thing is that I think the version they deprecated was more straightforward...